Library/ViewModels/LiveStreamContainerViewModel.swift

```diff
@@ -199,8 +199,13 @@ LiveStreamContainerViewModelInputs, LiveStreamContainerViewModelOutputs {
         .take(first: 1)
     )
 
-    self.projectImageUrl = project
-      .map { URL(string: $0.photo.full) }
+    self.projectImageUrl = Signal.merge(
+      configData.mapConst(nil),
+      Signal.combineLatest(
+        project.map { URL(string: $0.photo.full) },
+        configData.ignoreValues()
+      ).map(first)
+    )
```

Comment:
i think we can just re-express as:

```swift
self.projectImageUrl = Signal.merge(
  self.viewDidLoadProperty.signal.mapConst(nil),
  project.map { URL(string: $0.photo.full) }
)
```

?

also down to just remove the placeholder image in the storyboard.